packages/node_modules/@webex/webex-core/src/lib/services/services.js

```diff
@@ -145,6 +147,12 @@ const Services = WebexPlugin.extend({
     return catalog.markFailedUrl(url, noPriorityHosts);
   },
 
+
+  _setAllServiceUrls(serviceUrls) {
```

Comment:
a minor thing, but I feel the name of this method is a bit misleading - it's called "set" but it doesn't replace the values for all existing `_serviceUrls` - it just adds more

Reply:
changed it to updateServiceUrls